examples/with-tailwind/apps/docs/postcss.config.js

```diff
@@ -0,0 +1,11 @@
+// If you want to use other PostCSS plugins, see the following:
+// https://tailwindcss.com/docs/using-with-preprocessors
+
+module.exports = {
+  plugins: {
+    // Specifying the config is not necessary in most cases, but it is included
+    // here to share the same config across the entire monorepo
+    tailwindcss: { config: "../../tailwind.config.js" },
```

Comment:
If we created a `tailwind-config` package would it work to `require` it here and drop the `__dirname` from the `**/*.{js,ts,jsx,tsx}` stuff? Would that also succeed in isolating it?

Reply:
(Also, generally, I get nervous when these dependency things are tied together via path traversal, it's a refactoring hazard.)

Reply:
I hear you, fixed!